osquery/tables/events/darwin/network_events.cpp

```diff
@@ -0,0 +1,19 @@
+#include <ctime>
+#include <osquery/tables.h>
+#include <osquery/flags.h>
+#include <osquery/system.h>
+
+namespace osquery {
+namespace tables {
+
+QueryData genTable(QueryContext &context) {
+  Row r;
+
+  r["message"] = "Input from table generator!";
+
+  QueryData results;
+  results.push_back(r);
+  return results;
+}
+}
+}
```

Comment:
Add newlines at the end of files. You can configure your editor to do this automatically.

Reply:
Great, I’ll take care of the formatting issues. Any idea why my table isn’t available when I build? Also, is there a way to rebuild quickly after modifying the table spec files, such that the generated table c++ files get remade properly? Thanks!

Reply:
@chrisbennett-okta The table is not available because you have to specify on which platforms it has to be generated.
This can be done here https://github.com/osquery/osquery/blob/58ea43f53693d7fa95ed775f48c03c7b379a5baa/specs/CMakeLists.txt#L30 if it's a table that works on every platform, or here https://github.com/osquery/osquery/blob/58ea43f53693d7fa95ed775f48c03c7b379a5baa/specs/CMakeLists.txt#L67, specifying the platform.

When you modify the spec file, if you just try to build again, it will automatically updated the generated files and all the dependents.